create_rootfs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#Setup nfs server on Centos7
+
+# sudo yum install nfs-utils
+# sudo echo '$NFSROOT     192.168.0.1/24(rw,sync,no_root_squash,no_subtree_check)' >> /etc/exports
+# sudo exports -ra
+
+# sudo firewall-cmd --permanent --add-port=69/udp
+# sudo firewall-cmd --zone=public --add-service=tftp --permanent
+# sudo firewall-cmd --permanent --add-port=2049/tcp
+# sudo firewall-cmd --zone=public --add-service=nfs --permanent
+# sudo systemctl restart firewalld
+
+#Setup nfs server on Debian-like system
+
+sudo mkdir $NFSROOT
+sudo echo "$NFSROOT    192.168.0.1/24(rw,sync,no_root_squash,no_subtree_check)" >> /etc/exports
+sudo chown root:root $NFSROOT
+sudo chmod 755 $NFSROOT
+
+apt-get install nfs-kernel-server nfs-common debootstrap
```

Comment:
@pkubaj `please` is just alias of `sudo`, what is `doas` ? We typically use Debian-centric standard  syntax. Google `sudo apt-get" give me >800k results, `please apt-get` give me >1k results. Let's stick with 99% of population.